specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/examples/RedisCacheAccessPolicyAssignmentList.json

```diff
@@ -0,0 +1,44 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "cacheName": "cache1",
+    "api-version": "2023-05-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Cache/redis/cache1/accessPolicyAssignments/accessPolicy1",
+            "name": "cache1/accessPolicy1",
```

Comment:
The "name" of a nested resource is just the name of the resource, not the concatenated name of the parent + child.  In this case it should be "accessPolicy1".

Reply:
ok